extensions/grub.sh

```diff
@@ -140,25 +140,25 @@ pre_umount_final_image__900_export_kernel_and_initramfs() {
 }
 
 configure_grub() {
-	[[ -n "$SERIALCON" ]] \
-		&& GRUB_CMDLINE_LINUX_DEFAULT+=" console=${SERIALCON}"
+	[[ -n "$SERIALCON" ]] &&
+		GRUB_CMDLINE_LINUX_DEFAULT+=" console=${SERIALCON}"
 
-	[[ "$BOOT_LOGO" == "yes" || "$BOOT_LOGO" == "desktop" && "$BUILD_DESKTOP" == "yes" ]] \
-		&& GRUB_CMDLINE_LINUX_DEFAULT+=" quiet splash plymouth.ignore-serial-consoles i915.force_probe=* loglevel=3" \
-		|| GRUB_CMDLINE_LINUX_DEFAULT+=" splash=verbose i915.force_probe=*"
+	[[ "$BOOT_LOGO" == "yes" || "$BOOT_LOGO" == "desktop" && "$BUILD_DESKTOP" == "yes" ]] &&
+		GRUB_CMDLINE_LINUX_DEFAULT+=" quiet splash plymouth.ignore-serial-consoles i915.force_probe=* loglevel=3" ||
+		GRUB_CMDLINE_LINUX_DEFAULT+=" splash=verbose i915.force_probe=*"
```

Comment:
I don't think shellcheck like this is better. I think we should put the logical symbol in the front, otherwise it's a litter hard to known which line will execute.

Reply:
Hmm. This is (I think) `shellfmt`'s defaults applying here.
I understand it might not be optimal, but, at least it's consistent across the codebase.
Shortcircuits like this are _never_ easy to read, and they cause problems by exiting non-0 with `set -e`.
So those are better rewritten as normal `if`/`else`/`fi` clause, almost 100% of the times.
Shortcircuits = pain